shared/libraries/opcuatms/opcuatms_client/src/objects/tms_client_device_impl.cpp

```diff
@@ -360,7 +388,7 @@ void TmsClientDeviceImpl::findAndCreateStreamingOptions()
     }
     catch (const std::exception& e)
     {
-        LOG_E("Failed to find 'StreamingOptions' OpcUa node: {}", e.what());
+        LOG_W("Failed to find 'StreamingOptions' OpcUA node on OpcUA client device \"{}\": {}", this->localId, e.what());
```

Comment:
I'm not 100% on if all these logs should be warnings. I think some are okay to just be debug messages, as getting the default values is fine. 

Eg. any setter that in the core also just returns OPENDAQ_IGNORED should log it at the debug or info level. Similarly for example `getName` could just return the local ID and log at debug level, that the OPC UA call failed.